includes/sanitizers/class-amp-img-sanitizer.php

```diff
@@ -401,7 +401,8 @@ private function adjust_and_replace_nodes_in_array_map( $node_lists ) {
 	 */
 	private function adjust_and_replace_node( Element $node ) {
 		if ( $this->args['native_img_used'] || ( $node->parentNode instanceof Element && Tag::PICTURE === $node->parentNode->tagName ) ) {
-			$attributes = $this->maybe_add_lightbox_attributes( [], $node ); // @todo AMP doesn't support lightbox on <img> yet.
+			$node_attributes = AMP_DOM_Utils::get_node_attributes_as_assoc_array( $node );
```

Comment:
Why are the `$node_attributes` being obtained here and passed into `maybe_add_lightbox_attributes`? Since all of these attributes are already on `$node`, it's going to be overwriting many of the same attributes when it does this below:

```php
			foreach ( $attributes as $name => $value ) {
				$node->setAttribute( $name, $value );
			}
```

Reply:
I didn't realize the fact that attribute is present in the node until see the correction here - https://github.com/ampproject/amp-wp/pull/7178#discussion_r930416496

So I am reverting the changes here now. Thanks!